### DISCUSSION TITLE
Freeplane 1.11.13 not starting: JRE required that is installed

### DISCUSSION THREAD

**Discussion by julisokay** [freeplane/freeplane#1814] [Help] [answered]
Hi, 

im having trouble launching Freeplane, any tips would be highly appreciated.

Freeplane version: 1.11.13 git-revision=b6def874529c5a202021acbf83dbffd5d011dd76 dpolivaev 2024-05-01T15:02:37Z
JRE / JDK version: 21.0.2 from https://jdk.java.net/21/
 
This is the output of freeplaneConsole together with java -version to confirm a suitable JRE is installed. 

![grafik](https://github.com/freeplane/freeplane/assets/102518581/4616046e-b9a1-4692-8495-96746b33f587)

**Comment by HaaiHenkie** [comment, marked as answer]
Hi @julisokay,

Which Windows installer dit you use? If you use [Freeplane-Setup-with-Java-1.11.13.exe](https://sourceforge.net/projects/freeplane/files/freeplane%20stable/Freeplane-Setup-with-Java-1.11.13.exe/download) it should not give you any JRE troubles.

Cheers,

Henk

**Reply by julisokay** [reply]
Thanks @HaaiHenkie, this worked flawlessly.

**Comment by dpolivaev** [comment]
Check https://docs.freeplane.org/getting-started/Distributions.html?highlight=freeplane_java_home#distributions-for-microsoft-windows

**Reply by julisokay** [reply]
I can't mark your response as answer aswell, since this worked too. Since the other option using the Setup with java included is more beginner-friendly, i opted to select that as answer. Thanks a lot for your response anyway!
